docs/pages/agents/core-messaging/create-a-client.mdx

```diff
@@ -108,10 +108,11 @@ type ClientOptions = {
    * Add a client app version identifier that's included with API requests.
    * Production apps are strongly encouraged to set this value.
    *
-   * For example, you can use the following format: `appVersion: 'APP_NAME/APP_VERSION'`.
+   * You can use the following format: `appVersion: 'AGENT_NAME-agent/AGENT_VERSION'`.
+   * For example, `appVersion: 'alix-agent/2.x'`
    *
-   * Setting this value provides telemetry that shows which apps are using the
-   * XMTP client SDK. This information can help XMTP developers provide app
+   * Setting this value provides telemetry that shows which agents are using the
+   * XMTP client SDK. This information can help XMTP core developers provide you with agent
```

Comment:
can we remove `agent` here since this also applies to inbox apps?

Reply:
Ah, @rygine - I updated the text to be specific to agents b/c this is in the agents section. This guidance is also in the inboxes section, and there it refers to apps. 

However, it sounds like this is not helpful 😅, and it should always refer to apps, even when building an agent? And as you suggested above, we provide both agent and app examples?

Reply:
ahh, that's my bad! for some reason, i thought this was a shared page. feel free to use the agent specific wording.